modules/eco-configuring-machine-health-check-with-self-node-remediation.adoc

```diff
@@ -6,7 +6,7 @@
 [id="configuring-machine-health-check-with-self-node-remediation-operator_{context}"]
 = Configuring machine health checks to use the Self Node Remediation Operator
 
-Use the following procedure to configure the machine health checks to use the Self Node Remediation Operator as a remediation provider.
+Use the following procedure to configure the worker and control-plane machine health checks to use the Self Node Remediation Operator as a remediation provider.
```

Comment:
Maybe it would be better to use "or" instead of "and" as you use below.

Use the following procedure to configure the worker **or** control-plane machine health checks to use the Self Node Remediation Operator as a remediation provider.

Reply:
> 

Thanks @prabinovRedhat 

You are correct. I have changed the text to use "or" instead of "and" as I used below.